sdk/streamlib-python-wheel/tests/audio_window_probes.py

```
@@ -0,0 +1,89 @@
+"""Probes whose audio input port declares a window contract.
+
+The consumer the contract exists for: a model that wants exact-size blocks at
+its own rate gets them as ordinary user code, with no private buffering. Every
+one of these runs in its own child process, so what they report is the child's
+own stage — the same Rust code the parent's mailboxes run — honouring a contract
+that crossed the wiring envelope.
+"""
+
+import json
+
+from streamlib import AudioBlock, AudioWindowContract, input, log, processor
```

Ruff (0.16.2)

[error] Import `input` is shadowing a Python builtin

(A004)